Automattic/jetpack
# Blocks: update documentation

## Proposed changes:

This PR:

- updates the existing documentation around blocks to be up to date with recent changes in the monorepo, as well as current development tools in all our environments.
- adds a new section about API endpoints, following internal discussions.

### Other information:


## Jetpack product discussion

* p9dueE-7pi-p2#comment-9779

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Check for typos
* Check that info is correct.
